File: C++/sumofN.cpp
#include<iostream>
using namespace std;
int main()
{
    int n;
    int sum=0;
    cout<<"Enter the Number: ";
    cin>>n;
    for(int i=1;i<=n;i++)
    {
        sum=sum+i;
    }
    cout<<"Sum of Number upto "<<n<<" is "<<sum;
    return 0;
}

File: Java/sumofN.java
import java.util.*;
public class sumofN {
    public static void main(String[] args) {
        try (Scanner sc = new Scanner(System.in)) {
            System.out.print("Enter the Number: ");
            int n = sc.nextInt();
            int sum = 0;
            for (int i = 1; i <= n; i++) {
                sum = sum + i;
            }
            System.out.println(sum);
        }
    }
}
